### ISSUE TITLE
Scenario analysis application is not showing all data

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/emikat#30]
This is what I see on https://csis.myclimateservice.eu/study/35/step/1528/view/scenario-analysis

![image](https://user-images.githubusercontent.com/2961790/66201791-667cf100-e6a4-11e9-92e3-b5b9776db35b.png)

Error message seems to be bogus.

**Comment by DenoBeno** [comment]
If that's a "normal" behaviour, so be it. But I don't understand it.

@robandgo, please explain.

**Comment by RobAndGo** [comment]
I wasn't able to see what problem you are all referring to, but I do know that there are cases where for a given grid point, specific heat wave events may not occur. e.g. it is possible that for a rcp85 scenario, a rare or occasional event may occur, but a frequent event is not registered because in the statistics, this occurred at an interval of either much greater than once per year, or much lower than once per year.

Just a reminder of the event definitions:
frequent: 1 event per year
occasional: 1 event in 5 years
rare: 1 event in 20 years.

For the 30-year periods on which this is based, these values correspond to:
frequent: 30 events per 30-year period
occasional: 6 events per 30-year period
rare: 1.5 events per 30-year period.

Using these strict definitions lead to many events being missed, so I searched for ranges to these values, namely:
frequent: 25-45 events per 30-year period
occasional: 4-8 events per 30-year period
rare: 1-3 events per 30-year period

However, there were still cases where events weren't present, and at the time I wasn't sure how significant a problem this would be. I could adjust the search ranges above such that every grid point has an event.

**Comment by p-a-s-c-a-l** [comment]
API Call with missing Events: https://service.emikat.at/EmiKatTst/api/scenarios/3189/feature/view.2994/table/data

**Comment by RobAndGo** [comment]
I'll upload the file again (I initially uploaded the rcp26 file for some reason):
[europe_rcp45_20110101-20401231.csv.txt](https://github.com/clarity-h2020/emikat/files/3885868/europe_rcp45_20110101-20401231.csv.txt)

**Comment by RobAndGo** [comment]
After discussing the issue further with @humerh, and seeing that @humerh did have all the data already correctly imported into Emikat, we determined that the problem is that which I addressed in the [post above](https://github.com/clarity-h2020/emikat/issues/30#issuecomment-539884088). That is, it is normal that a grid point may not have all three types of events (frequent, occasional, rare) for a particular RCP-scenario/future time period.

E.g. for the grid point at latitude:32.215336 and longitude: -4.425540, the rcp45, 2071-2100 does not have a rare event, but a rare event does exist for the rcp45 2011-2040 period.

```
latitude;longitude;urban area;country;RCP;Period;Tx95p;Frequent(1.0);Occasional(0.2);Rare(0.05)
32.215336;-4.425540;;;rcp45;20710101-21001231;36.5;40_3.533d;44.5_0.300d;
32.215336;-4.425540;;;rcp45;20110101-20401231;36.5;38.5_3.866d;42.5_0.366d;43_0.066d

```
